unittests/resources/checks/frontend_checks.py

```diff
@@ -12,6 +12,7 @@
 from reframe.core.exceptions import ReframeError, PerformanceError
 
 
+@rfm.extend_test
```

Comment:
No need for extension in any of these tests. They should be adapted to use the hooks.